sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -136,3 +136,17 @@ export const OFFERING_PROGRAM_YEAR_MISMATCH = "OFFERING_PROGRAM_YEAR_MISMATCH";
  */
 export const OFFERING_DOES_NOT_BELONG_TO_LOCATION =
   "OFFERING_DOES_NOT_BELONG_TO_LOCATION";
+/**
+ * The text content to perform the application withdrawal is not in the
+ * expected format and cannot be parsed.
+ */
+export const APPLICATION_WITHDRAWAL_TEXT_CONTENT_FORMAT_ERROR =
```

Comment:
IMO line 143 and 151 must belong to same type of error.

Reply:
No, I believe, those are 2 different types of errors, both are badrequest exceptions but one defines the error for header and footer and the other is for the content data type. The difference is, there is a possibility even if the data type errors are warning, the business validation can be proceeded, but for the header and footer validation, the processing of the file needs to be terminated.

Reply:
But as discussed both errors will be shown in the table not in the toast.